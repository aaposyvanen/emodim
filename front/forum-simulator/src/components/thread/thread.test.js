import React from 'react';
import renderer from 'react-test-renderer';
import Thread from "./thread";

describe("Thread", () => {
    it("renders correctly", () => {
        const component = renderer.create(
            <Thread />,
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
