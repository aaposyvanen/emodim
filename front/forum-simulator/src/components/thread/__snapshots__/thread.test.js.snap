// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thread renders correctly 1`] = `
<div>
  <div
    class="thread"
  >
    <div
      class="thread-changer"
    >
      <button
        class="thread-change-button"
      >
        Prev
      </button>
      <div
        class="current-thread-label"
      >
        1
      </div>
      <button
        class="thread-change-button"
      >
        Next
      </button>
    </div>
    <div
      class="title"
    >
      Thread title
    </div>
    <div
      class="message-area"
    >
      <div
        class="thread-start"
      />
      <div
        class="comments"
      />
    </div>
  </div>
</div>
`;
